easybuild/easyblocks/a/abaqus.py

```diff
@@ -126,10 +130,9 @@ def install_step(self):
             # rather than a regular dictionary (where there's no guarantee on key order in general)
             std_qa = OrderedDict()
 
-            # Enable Extended Product Documentation
-            std_qa[selectionstr % (r"\[ \]", "Extended Product Documentation")] = "%(nr)s"
-            # Enable Abaqus CAE (docs)
-            std_qa[selectionstr % (r"\[ \]", "Abaqus CAE")] = "%(nr)s"
+            # Disable Extended Product Documentation (because it has a troublesome Java dependency)
+            std_qa[selectionstr % (r"\[*\]", "Extended Product Documentation")] = "%(nr)s"
+            installed_docs = False
```

Comment:
I don't fully follow here, now it looks like there's no way to install docs (`installed_docs` is always false) even if someone satisfies the Java dependency

Reply:
Can't you have a conditional path here based on the availability of Java?

Reply:
Correct, there is no way to install the docs. We'd have to check for the type of Java, to check if something has gone out of their way to replace Java/11.x.x with the Oracle version or an adoptjdk-variant.
This easyblock in the repo is currently broken and can no longer install ABAQUS-2020.eb or 2022.

I considered just removing all the code regarding docs completely, but left it blocked by this variable just in case someone really wanted to do all those external changes to use the docs again.